src/Blog_App_frontend/src/AddBlog.jsx

```diff
@@ -27,11 +34,13 @@ function AddBlog({ handleAddBlog }) {
         validationSchema={addBlogSchema}
         onSubmit={async (values, { setSubmitting, resetForm }) => {
           try {
-            await handleAddBlog(values);
+            await mutation.mutateAsync(values);
```

Comment:
why mutateAsync instead of mutate?

Reply:
Asynchronous ayitt blog add akkan
For flexible flow